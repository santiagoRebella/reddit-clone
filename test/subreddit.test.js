const React = require('react');
const Subreddit = require('components/subreddit');
const renderer = require('react-test-renderer');

jest.mock('react-router-dom', () => ({
  NavLink: 'NavLink'
}))

test('Subreddit is rendered correctly', () => {
  const component = renderer.create(
    <Subreddit
        key="someid"
        title="Test title"
        description="Test Description"
        url="Test Url"
        image="Test image"
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
