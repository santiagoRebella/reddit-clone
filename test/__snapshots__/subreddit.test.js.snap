// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subreddit is rendered correctly 1`] = `
<div
  className="card"
  style={
    Object {
      "display": "inline-block",
      "margin": 10,
    }
  }
>
  <div
    className="card-body"
  >
    <h5
      className="card-title"
    >
      Test title
    </h5>
    <NavLink
      className="btn btn-primary"
    />
  </div>
</div>
`;
